src/manager/tilingManager.js

```diff
@@ -141,7 +141,7 @@ var TilingManager = class TilingManager extends MsManager {
                 }
                 let layout = msWorkspace.tilingLayout;
 
-                layout.onTile();
+                layout.tileAll();
```

Comment:
It was throwing an error (onTile not found) when changing layouts, maybe it can be simply discarded?